src/Core/Infrastructure/RealTime/Repository/Acknowledgement/DbFactoryAcknowledgement.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/*
+ * Copyright 2005 - 2021 Centreon (https://www.centreon.com/)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * For more information : contact@centreon.com
+ *
+ */
+declare(strict_types=1);
+
+namespace Core\Infrastructure\RealTime\Repository\Acknowledgement;
+
+use Core\Domain\RealTime\Model\Acknowledgement;
+use Core\Infrastructure\Common\Repository\DbFactoryUtilitiesTrait;
+
+class DbFactoryAcknowledgement
```

Comment:
this class and file should be name DbAcknowledgementFactory to be consistent with your other factory